src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -21,12 +21,13 @@ public Node(Node<T> prev, T item, Node<T> next) {
 
     @Override
     public void add(T value) {
+        Node<T> newNode = new Node(tail, value, null);
         if (head == null) {
-            head = tail = new Node<>(null, value, null);
+            head = tail = newNode;
```

Comment:
```suggestion
            head = newNode;
```
if you moved it out, why it's still here?)

Reply:
ой,нечайно після провірки тестів забагато ctrl-z переклацнула, перепрошую))